### PR TITLE
Escape comma

### PR DESCRIPTION
**Pull request by alechirsch**
I had a use case where I needed to filter on items that included a comma. I added some logic to support escaping the comma character.

### CODE REVIEW

**Comment by alechirsch** [comment]
Getting the timeout failure again...

**Comment by alechirsch** [comment]
@jamesdixon Do you have time to look at this?

**Comment by jamesdixon** [comment]
@alechirsch unfortunately, i do not. Right in the middle of beginning the beta phase for our product. I'll take a look when I can!

**Comment by jamesdixon** [comment]
@alechirsch i'd be more than happy to add you as a contributor if you'd be interested

**Comment by alechirsch** [comment]
@jamesdixon I am interested, I have never been a contributor for an open source package before though. Would that give me publishing privileges to npm as well?

**Comment by jamesdixon** [comment]
@alechirsch initially, it would just give you the ability to manage issues, merge PRs, etc.

**Comment by alechirsch** [comment]
@jamesdixon Sounds good to me

**Comment by alechirsch** [comment]
@jamesdixon Any updates for this?

**Comment by jamesdixon** [comment]
@alechirsch sorry for the delay. Added you as a collaborator

**Comment by alechirsch** [comment]
@jamesdixon are you good with this PR?

**Comment by jamesdixon** [comment]
@alechirsch are you talking about the use case for something like 

`?filter[status]=scheduled,late`

?

**Comment by alechirsch** [comment]
This is the use case where you have a comma in the data itself, for example if you had a row with the data "city, state", what you send across is "city\\\\\\\\,state"

**Comment by jamesdixon** [comment]
Ok, thanks. Would you mind adding a test for this specific case?

**Comment by alechirsch** [comment]
@jamesdixon Sure thing, I have been very busy recently. Not sure when I will get it in.

**Comment by alechirsch** [comment]
@jamesdixon Finally got around to adding a test for this.

File: src/index.js [hunk 1]
```diff
@@ -382,9 +391,17 @@ export default (Bookshelf, options = {}) => {
                                 // Remove all but the last table name, need to get number of dots
                                 key = internals.formatRelation(internals.formatColumnNames([key])[0]);
 
-                                // Determine if there are multiple filters to be applied
-                                value = value.toString().indexOf(',') !== -1 ? value.split(',') : value;
-                                qb.whereIn(key, value);
+
+                                if (_isNull(value)){
```
Comment (alechirsch) [inline src/index.js:398]: This is the fix for filtering on null

**Comment by alechirsch** [comment]
@jamesdixon Should I go ahead and merge this? Would you be able to release to npm?

**Comment by jamesdixon** [comment]
@alechirsch my apologies for the delayed reply. Things have been very busy for me and it's honestly been difficult to divert attention to my open source projects.

That said, feel free to merge this and the other PRs. I'd recommend we label this as a beta given that there have been a lot of changes. Once you've merged, I can publish to NPM and start testing in my own project.

Thanks for all of your hard work!

James

**Comment by alechirsch** [comment]
@jamesdixon Great, I agree this should be kept labeled as beta.

**Comment by jamesdixon** [comment]
@alechirsch would you mind updating the changelog and close out any bugs that these PRs resolved? I can publish to NPM once that's done. Thanks!

**Comment by alechirsch** [comment]
@jamesdixon I closed related issues. The change log does not have anything for beta1, was there a separate place where you put that or was it not added.

**Comment by jamesdixon** [comment]
@alechirsch thanks. It probably wasn't added, but I wouldn't worry about it at this point 👍

**Pull request merged**